tao_compiler/mlir/disc/transforms/fusion_utils.h

```diff
@@ -202,9 +205,13 @@ class FusionPattern {
   SmallVector<Operation*, 4>& getRootOps() { return root_ops_; }
 
   // Returns values that are outputs of any lmhlo op in the fused pattern and
-  // are only consumed by the lmhlo ops inside the fused pattern.
+  // are only consumed by the lmhlo ops outside the fused pattern.
   FusionValueList& getInternalResults() { return internal_results_; }
 
+  // Returns values that are outputs of any lmhlo op in the fused pattern and
+  // are only consumed by the lmhlo ops inside the fused pattern.
+  FusionValueList& getExternalOnlyResults() { return external_only_results_; }
```

Comment:
Is the comments being mistakenly upside-down?
outside -> Internal ,  inside -> External ?

Reply:
Internal means inside. It is different from inter v.s. intra.